test/integration/start-integration-env.sh

```diff
@@ -83,10 +90,9 @@ sleep 15
 docker exec ${CONTAINER_NAME} bash -c ". /test/integration/vagrantenv.sh; cd /sifnode; SMART_CONTRACTS_DIR=/smart-contracts python3 /test/integration/peggy-basic-test-docker.py /network-definition.yml"
 docker exec ${CONTAINER_NAME} bash -c '. /test/integration/vagrantenv.sh; cd /sifnode; SMART_CONTRACTS_DIR=/smart-contracts python3 /test/integration/peggy-e2e-test.py /network-definition.yml'
 
-# Uncomment these tests in PR 378
 # Rebuild sifchain, but this time don't use validators
 
-#sudo rm -rf $NETWORKDIR && mkdir $NETWORKDIR
-#ADD_VALIDATOR_TO_WHITELIST= bash ${BASEDIR}/test/integration/setup_sifchain.sh && . $envexportfile
-#
-#docker exec ${CONTAINER_NAME} bash -c ". /test/integration/vagrantenv.sh; cd /sifnode; SMART_CONTRACTS_DIR=/smart-contracts python3 /test/integration/no_whitelisted_validators.py /network-definition.yml"
+sudo rm -rf $NETWORKDIR && mkdir $NETWORKDIR
```

Comment:
Does this have to be sudo? Sudo's in this script are very annoying.

Reply:
I doubt it.  That's from the original scripts; there was a note saying it needed sudo, but I think it's not true.

Reply:
Removed.

Reply:
Turns out it does break github actions.  Putting it back.